src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  })
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+})
+
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('mousedown', (e) => {
+    if(e.buttons === 1 || e.buttons === 2) {
```

Comment:
property is called `button` https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button